Add party management service to the services page, targeting #8027. (#8075)
digital-asset/daml: docs/source/app-dev/services.rst

Utility services
****************

.. _party-service:

Party management service
========================

Use the **party management service** to allocate parties on the ledger and retrieve information about allocated parties.

Allocating parties is necessary to interact with the ledger. For more information, refer to the pages on :doc:`Identity Management</concepts/identity-and-package-management>` and :ref:`the API reference documentation <com.daml.ledger.api.v1.admin.PartyManagementService>`.

> 

Author: ```suggestion
```

Utility services
****************

.. _party-service:

Party management service
========================

Use the **party management service** to allocate parties on the ledger and retrieve information about allocated parties.

> Allocating parties is necessary to interact with the ledger. See also our :doc:`documentation on Identity Management</concepts/identity-and-package-management>`.

Reviewer: ```suggestion
Allocating parties is necessary to interact with the ledger. For more information, refer to the pages on :doc:`Identity Management</concepts/identity-and-package-management>` and :ref:`the API reference documentation <com.daml.ledger.api.v1.admin.PartyManagementService>`.
```

> Party management service offers both retrieving details on an individual party and listing all known parties.

Reviewer: ```suggestion
```

> For full details, see :ref:`the proto documentation for the service <com.daml.ledger.api.v1.admin.PartyManagementService>`.

Reviewer: ```suggestion
```
